README.md

```diff
@@ -27,20 +27,26 @@ We use GitHub Issues to organize our work.  Here's some quick reading about [how
-7. Install all necessary node packages with `npm install`. This might take a few minutes.
-8. Now you can try out the current UI or play with the code!
-9. To run a local version of the site so you can play around or debug your changes, type `npm start`. It will automatically open a browser window and run the code locally. If you make changes to a file and save it, the page will automatically update and use the new code. Would also recommend getting the [React Developer Tools](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=4&cad=rja&uact=8&ved=0ahUKEwiZ__6Vg_jVAhWQ14MKHczrDtoQFgg4MAM&url=https%3A%2F%2Fchrome.google.com%2Fwebstore%2Fdetail%2Freact-developer-tools%2Ffmkadmapgofadopljbjfkapdkoienihi%3Fhl%3Den&usg=AFQjCNEv0udXgBoaukzJa59I_vufhScUbQ) extension for Chrome.
-10. `Ctrl-C` in the terminal to stop the app when it's running.
-
-To keep up to date with the latest changes to the `dev` branch, you'll need to use git to add a new remote branch with this repo's address. The page about [git workflow](https://github.com/codeforboston/cliff-effects/wiki/Git-Workflow) includes some notes about that.
+To follow these instructions, it'd help if you:
+- have mentally perpared for experimentation and learning and/or
+- know how to open your command prompt/terminal
+- know a bit about how to navigate to different folders in your command prompt/terminal
+- know a bit about installing software for your command prompt/terminal, like nodejs
+
+Instructions:
+1. If you know how, fork and clone the repo, and create a link (a remote branch) to our repo. If not, see this [github guide](https://help.github.com/articles/fork-a-repo/). Go through all the steps.
+2. In your termainal, do `git checkout dev`. *Do not touch the `master` branch.*
+3. If you don't have nodejs, install it (see [install nodejs](https://nodejs.org/en/)). If you're not sure if you have nodejs, do the next step. If you don't get a number, you don't have nodejs.
+4. Check your node.js version in your terminal with `node --version`. It should show a number. Version 5.0 or greater is preferred.
+5. In your command prompt/terminal, do `npm install`. This might take a few minutes.
+6. Now you'll be able to try out the current UI or play with the code!
+7. To see a local version of the site, in your command prompt/terminal type `npm start`. It should open a browser window with the site loaded. It works best in Chrome.
+8. To make changes to the site and see them, just change the code in the files, save the changes, and the browser page will automatically reload. If you don't see the changes, refresh the page. If it's still not looking different, you may have done something incorrectly. Feel free to ask one of us.
+9. To stop the local version, in your command prompt/terminal press `Ctrl-C`.
```

Comment:
Suggested change:
> `ctrl+c` to stop the local version.

Reply:
I think what I was trying to do was clarify that they don't type 'ctrl+c' into their terminal. Maybe:

> 9. Press ctrl+c to stop the local version.

Addressing a different issue of wording that might also help educate:

> 9. Press cltr+c to stop the local server that you have running.

I know it's longer, and I'm ok with either version. The second one just might help them better understand what's going on and give them an easier time looking up answers to future questions. It may be too complex to address here, though.

Reply:
You're right - unfair assumption on my part.  Let's stick with "press," and go with the longer version as you have suggested.